### PR TITLE
Show nested form label and nested form entry index.

### PR DESCRIPTION
**Pull request by niuyunsi**

### CODE REVIEW

File: src/components/NestedForm/index.ts [hunk 1]
```diff
@@ -9,7 +9,10 @@ const FieldTemplate: IField = {
     id: '',
     label: 'Detail',
     type: Type,
-    fields: Array<IField>()
+    fields: Array<IField>(),
+    options: {
+        showIndex: true
```
Comment (linweigao) [inline src/components/NestedForm/index.ts]: uiOptions: {
showIndex:true;
}

Otherwise, server cannot save the value.

**Comment by linweigao** [comment]
Shall we have an option to disable the delete button?

**Review by linweigao — APPROVED** [review]

File: src/components/NestedForm/index.ts (hunk 1 above)
Comment (niuyunsi) [reply src/components/NestedForm/index.ts]: Done

File: src/components/NestedForm/index.ts [hunk 2]
```diff
@@ -9,7 +9,13 @@ const FieldTemplate: IField = {
     id: '',
     label: 'Detail',
     type: Type,
-    fields: Array<IField>()
+    fields: Array<IField>(),
+    options: {
+        uiOptions: {
+            showIndex: true,
+            showDeleteBtn: true
```
Comment (linweigao) [inline src/components/NestedForm/index.ts:16]: default should be false?

**Pull request merged**